http/server.go

```diff
@@ -54,8 +54,8 @@ func NewServer[I core.BlockId, R core.BlockIdRef[I]](store core.BlockStore[I], s
 	server.http = &http.Server{
 		Addr:           serverConfig.Address,
 		Handler:        mux,
-		ReadTimeout:    10 * time.Second,
-		WriteTimeout:   10 * time.Second,
+		ReadTimeout:    100 * time.Second,
```

Comment:
I have another issue about fine tuning timeouts.  Just brute force bumping for now.